packages/transition-common/src/services/line/__tests__/Line.test.ts

```
@@ -229,3 +233,61 @@ describe('Test schedules management', () => {
+            true,
+            collectionManager
+        );
+        return { line, path1, path2 };
+    };
+
+    test.each([
+        [540.7, 541],
+        [-1, null],
+        [null, null]
+    ])('stores routed duration %p as %p', async (duration, expected) => {
+        const consoleError = jest.spyOn(console, 'error').mockImplementation(() => {});
```

ESLint

[error] Unexpected empty arrow function.

(no-empty-function)